canvas_modules/harness/src/client/App.js

```diff
@@ -1639,8 +1639,10 @@ export default class App extends React.Component {
 			{ action: "arrangeVertically", label: "Arrange Vertically", enable: true }
 		];
 
-		const notificationConfig = { action: "notification", label: "Notifications", enable: true, notificationHeader: "Notifications" };
-		const notificationConfig2 = { action: "notification", label: "Notifications", enable: true, notificationHeader: "Notifications Canvas 2" };
+		const notificationConfig = { action: "notification", label: "Notifications", notificationHeader: "Notification Center", notificationSubtitle: "subtitle status", enable: true,
+			emptyMessage: "You don't have any notifications right now.", clearAllMessage: "Clear all" };
+		const notificationConfig2 = { action: "notification", label: "Notifications", notificationHeader: "Notification Center Canvas 2", enable: true,
+			emptyMessage: "You don't have any notifications right now.", clearAllMessage: "Clear all" };
```

Comment:
We will need to document the additions to notificaiton config in the configs page of the Elyra-canvas wiki.